app/uk/gov/hmrc/customs/notification/services/customsNotificationServices.scala

```diff
@@ -102,47 +101,49 @@ class CustomsNotificationRetryService @Inject()(
       }
   }
 
```

Comment:
misleading function name - how about something like `saveNotificationToDatabaseAndPushOrPullIfNotAnyPF`.
Also ragged formatting of parameters - please align